src/test/java/bio/terra/common/logging/LoggingTestController.java

```diff
@@ -55,4 +55,18 @@ public void testStructuredLogging() throws JsonProcessingException {
     // The GSON should look like {"foo": {"bar": "baz"}}
     LOG.info("GSON object", jsonObject);
   }
+
+  @GetMapping("/testAlertLogging")
+  public void testAlertLogging() throws JsonProcessingException {
```

Comment:
I would split your two test cases into separate endpoints and test methods.

Reply:
Done